src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  let partner = (<p className="Person__partner">I am not married</p>);
+
+  if (isMarried && sex === 'f') {
+    partner = (<p className="Person__partner">{`${partnerName} is my husband`}</p>);
+  }
+
+  if (isMarried && sex === 'm') {
+    partner = (<p className="Person__partner">{`${partnerName} is my wife`}</p>);
+  }
```

Comment:
use Conditional (ternary) operator

Reply:
agree, do it with ternary operator
You don't need condition for all tag `p`, you need it only for text inside this tag